src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -100,11 +100,64 @@ static std::string          subscribeContextAvailabilityCollectionName;
 static Notifier*            notifier;
 static bool                 multitenant;
 
+thread_local std::vector<ContextElementResponse*>  savedCerVector;
```

Comment:
Maybe change name savedCerVector -> toBeFreedCerVector ?

("saved" does not provide hints about the purpose of the vector)

Reply:
Fixed in dcf5197 (cerVectorForDelayedRelease)